api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -84,6 +98,110 @@ public void sendWelcomeEmail(final String contactEmail, final String password, f
+        String.format(
+            "User %s has passed the %.2f free tier dollar threshold.  Current total usage is $%.2f with remaining balance $%.2f",
+            user.getUsername(), threshold, currentUsage, remainingBalance);
+    log.info(logMsg);
+
+    if (workbenchConfig.featureFlags.sendFreeTierAlertEmails) {
+      final String msgHtml =
+          buildHtml(
+              FREE_TIER_DOLLAR_THRESHOLD_RESOURCE,
+              freeTierDollarThresholdSubstitutionMap(
+                  user, currentUsage, remainingBalance, expirationTimeIfKnown));
+      final String subject =
+          String.format(
+              "Reminder - %s Free credit usage in All of Us Researcher Workbench",
+              formatPercentage(threshold));
+
+      final MandrillMessage msg =
+          new MandrillMessage()
+              .to(Collections.singletonList(validatedRecipient(user.getContactEmail())))
+              .html(msgHtml)
+              .subject(subject)
+              .fromEmail(workbenchConfig.mandrill.fromEmail);
+
+      sendWithRetries(
+          msg, String.format("User %s passed a free tier dollar threshold", user.getUsername()));
+    }
+  }
+
+  @Override
+  public void alertUserFreeTierTimeThreshold(
+      final DbUser user,
+      long daysRemaining,
+      final LocalDate expirationDate,
+      double remainingDollarBalance)
+      throws MessagingException {
+    final WorkbenchConfig workbenchConfig = workbenchConfigProvider.get();
+
+    final String logMsg =
+        String.format(
+            "User %s has %d days remaining until their expiration date of %s.  Current total usage is $%.2f",
+            user.getUsername(), daysRemaining, formatDate(expirationDate), remainingDollarBalance);
+    log.info(logMsg);
+
+    if (workbenchConfig.featureFlags.sendFreeTierAlertEmails) {
+      final String msgHtml =
+          buildHtml(
+              FREE_TIER_TIME_THRESHOLD_RESOURCE,
+              freeTierTimeThresholdSubstitutionMap(
+                  user, daysRemaining, remainingDollarBalance, expirationDate));
+
+      final String subject =
+          String.format(
+              "Reminder - %d days remaining in Free credits in All of Us Researcher Workbench",
+              daysRemaining);
+
+      final MandrillMessage msg =
+          new MandrillMessage()
+              .to(Collections.singletonList(validatedRecipient(user.getContactEmail())))
+              .html(msgHtml)
+              .subject(subject)
+              .fromEmail(workbenchConfig.mandrill.fromEmail);
+
+      sendWithRetries(
+          msg, String.format("User %s passed a free tier time threshold", user.getUsername()));
+    }
+  }
+
+  @Override
+  public void alertUserFreeTierExpiration(final DbUser user) throws MessagingException {
+    final WorkbenchConfig workbenchConfig = workbenchConfigProvider.get();
+
+    final String logMsg =
+        String.format("Free credits have expired for User %s", user.getUsername());
+    log.info(logMsg);
```

Comment:
should this be info or a warning message

Reply:
I don't think this should be a warning because this is an expected situation, from our system's perspective.